prow/crier/controller.go

```diff
@@ -68,7 +69,10 @@ func New(
 		Named(fmt.Sprintf("crier_%s", reporter.GetName())).
 		For(&prowv1.ProwJob{}).
 		WithOptions(controller.Options{MaxConcurrentReconciles: numWorkers,
-			RateLimiter: workqueue.DefaultControllerRateLimiter()}).
+			RateLimiter: workqueue.NewMaxOfRateLimiter(
+				workqueue.NewItemExponentialFailureRateLimiter(5*time.Millisecond, 500*time.Second),
+				// 100 qps, 1000 bucket size.
+				&workqueue.BucketRateLimiter{Limiter: rate.NewLimiter(rate.Limit(100), 1000)})}).
```

Comment:
How do we know these are safe settings?

Also, what settings are we currently using with workqueue.DefaultControllerRateLimiter()?

Reply:
The `workqueue.DefaultControllerRateLimiter()` is calling `workqueue.NewMaxOfRateLimiter` internally with `rate.Limit` set to 10 and bucket size set to 100. I cannot say for sure that these are safe for sure, but we have example in the codebase that is using a higher `rate.Limit` and bucket size.

Reply:
> The workqueue.DefaultControllerRateLimiter() is calling workqueue.NewMaxOfRateLimiter internally with rate.Limit set to 10 and bucket size set to 100.

This sort of background information is exactly what should be going into the commit message or PR description.

If you're not feeling comfortable with the size of the increase, then it would be prudent to make the bump less drastic. For example, a 20% increase, followed by another 20% increase, etc. So you'd have to break this PR up into several smaller ones (and then measure the impact after each one after it's rolled out).

Reply:
I would say double up the QPS and bucket size instead of 20% increase, wdyt?

Reply:
SGTM, thanks.

Reply:
Updated and also included the background info in commit message. Thanks.